statik/database.py

```diff
@@ -418,6 +418,14 @@ def __init__(self, model=None, session=None, **kwargs):
                     "Attempting to look up primary keys for ManyToMany " +
                     "field relationship: %s", self.field_values[field_name]
                 )
+
+                # check if field values contains duplicates
+                if len(self.field_values[field_name]) != len(set(self.field_values[field_name])):
+                    logger.warning("Duplicates found in field value array: %s",
```

Comment:
It is possible to do the checking in `common.py`, which is where the yml is loaded in. However, this means it would be run everytime the yml is loaded in, instead of only being run when there is an array. Thoughts?